src/modules/uniplayer/UniplayerContainer.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import * as Redux from 'react-redux';
```

Comment:
Will uniplayer get anything from global state or is it more of a low-level tool?

Reply:
I was going to pass the player assets like the `url ID` and the `title` through the global redux state so the player could read it and update